public/app/plugins/panel/text/models.cue

```diff
@@ -0,0 +1,21 @@
+package grafanaschema
+
+Family: {
+    lineages: [
+        [
+            {
+                TextMode: "html" | "markdown" @cuetsy(targetType="enum")
+
+                PanelOptions: {
+                    mode: TextMode | *"markdown"
+                    content: string | *"""
+                    # Title
+
+                    For markdown syntax help: [commonmark.org/help](https://commonmark.org/help/)
+                    """
```

Comment:
nit/q: this seems like something that's more the responsibility of e.g. a placeholder attribute on an input tag, no?

Reply:
Yes -- but in this case we actually save that default into the panel model and people start with editing it.  Typically "placeholder" text is replaced 100% once you start typing.

Not saying it is great... but matches current behavior

Reply:
Another interesting case!